sharktuner/pyproject.toml

```diff
@@ -1,5 +1,9 @@
+[build-system]
```

Comment:
Changes to this file should go to a separate PR that enables nightly releases for sharktuner.